src/renderer/views/MissingActivationView.tsx

```diff
@@ -35,10 +35,10 @@ function MissingActivationView() {
   const history = useHistory();
   const account = useStore("account");
 
-  const { activated } = useActivation(false);
+  const { activated } = useActivationStatus();
```

Comment:
Isn't `useActivationStatus()` requires `usePolling` parameter?
You haven't specified on other `useActivationStatus()` usage too.

Reply:
`usePolling` parameter is optional, not an requirement.